pages/doc/wavefront_spring_boot3.md

```diff
@@ -0,0 +1,450 @@
+---
+title: Wavefront for Spring Boot 3
+keywords:
+tags:
+sidebar: doc_sidebar
+permalink: wavefront_springboot3.html
+summary: Examine Spring Boot 3 data in Wavefront dashboards and charts
+---
+
+Wavefront for Spring Boot allows you to quickly configure your environment, so Spring Boot 3 components send metrics, histograms, and traces or spans to the Wavefront service.
+
+## Features
+
+If you use Wavefront for Spring Boot:
+
+* A large number of metrics are exposed by default. See the [Spring Boot documentation on Supported Metrics](https://docs.spring.io/spring-boot/docs/3.0.x/reference/html/actuator.html#actuator.metrics.supported) for details.
+* You have tracing support for all the common Spring Boot components, such as Spring MVC, Spring Web, Spring Async, Feign, Hysterix, JMS, JDBC, Mongo, Zuul, Reactor, RxJava, Redis, Logging, Spring Messaging, and RabbitMQ.
+* The distributed tracing spans in your applications are automatically converted to the Wavefront span format.
+
+## Sending Data from Spring Boot into Wavefront
+
+You can send data from your Spring Boot applications into Wavefront using the Wavefront for Spring Boot Starter (all users) or the Wavefront Spring Boot integration (Wavefront customers and trial users).
+
+* **Wavefront for Spring Boot Starter**
+  <br/> If you configure your application with the Wavefront for Spring Boot starter, you can send metrics, histograms, and traces/spans to the Wavefront service. Once the data is in Wavefront, you can view your data, find hotspots, and gather more data.
+  - **Freemium** All users can run the Spring Boot Starter with the default settings to view their data in the Wavefront Freemium instance. Certain limitations apply, for example, alerts are not available.
```

Comment:
- **Freemium**: ...
- **Wavefront Customer or Trial User**: ...
- **Wavefront Spring Boot Integration**:  

Same comment applies to these bullets in the wavefront_spring_boot.md file.